src/content/docs/integrations/amazon-integrations/aws-integrations-list/aws-metric-stream.mdx

```diff
@@ -72,9 +72,11 @@ To stream CloudWatch metrics to New Relic you need to create Kinesis Data Fireho
 * If you manage multiple regions within those accounts, then each region needs to be configured with a different Kinesis Data Firehose pointing to New Relic.
 * You will typically map one or many AWS accounts to a single New Relic account.
 
-### Automated setup using CloudFormation [#setup-cloudformation]
+### Guided setup using CloudFormation [#setup-cloudformation]
 
-We provide a [CloudFormation template](https://console.aws.amazon.com/cloudformation/home?#/stacks/quickcreate?templateURL=https://nr-downloads-main.s3.amazonaws.com/cloud_integrations/aws/cloudformation/MetricStreams_CloudFormation.yml&stackName=NewRelic-Metric-Stream&param_NewRelicDatacenter=US) that automates this process. This needs to be applied to each AWS account and region you want to monitor in New Relic. 
+First, you need to link each of your AWS accounts with your New Relic account. To do so, follow the instructions in the AWS UI. 
```

Comment:
The instructions / wizard is on the New Relic > Infrastructure > AWS UI (not in AWS UI). 
We could also mention that, alternatively, the same step can be automated using NerdGraph API (link to https://docs.newrelic.com/docs/apis/nerdgraph/examples/nerdgraph-cloud-integrations-api-tutorial/#link-aws).